docs/install.md

```diff
@@ -0,0 +1,204 @@
+
+This guide explains how to install Tekton Dashboard. It covers the following topics:
+
+* [Before you begin](#before-you-begin)
+* [Pre-requisites](#pre-requisites)
+* [Which version should I use](#which-version-should-i-use)
+* [Installing Tekton Dashboard on Kubernetes](#installing-tekton-dashboard-on-kubernetes)
+* [Installing Tekton Dashboard on OpenShift](#installing-tekton-dashboard-on-openshift)
+* [Accessing the Dashboard on Kubernetes](#accessing-the-dashboard-on-kubernetes)
+* [Accessing the Dashboard on OpenShift](#accessing-the-dashboard-on-openshift)
+* [Uninstalling the Dashboard on Kubernetes](#uninstalling-the-dashboard-on-kubernetes)
+* [Next steps](#next-steps)
+
+## Before you begin
+
+1. Choose the version of Tekton Dashboard you want to install. You have the following options:
+
+   * **[Official](https://github.com/tektoncd/dashboard/releases)** - install this unless you have a specific reason to go for a different release.
+   * **[Nightly](../tekton/README.md#nightly-releases)** - may contain bugs, install at your own risk. Nightlies live at `gcr.io/tekton-nightly`.
+   * **[`HEAD`]** - this is the bleeding edge. It contains unreleased code that may result
+     in unpredictable behavior. To get started, see the [development guide](../DEVELOPMENT.md) instead of this page.
+
+2. If you don't have an existing Kubernetes cluster, set one up, version 1.15 or later:
+
+   ```bash
+   #Example command for creating a cluster on GKE
+   gcloud container clusters create $CLUSTER_NAME \
+     --zone=$CLUSTER_ZONE --cluster-version=1.15.11-gke.5
+   ```
+
+3. Grant `cluster-admin` permissions to the current user:
+
+   ```bash
+   kubectl create clusterrolebinding cluster-admin-binding \
+   --clusterrole=cluster-admin \
+   --user=$(gcloud config get-value core/account)
+   ```
+
+   See [Role-based access control](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#prerequisites_for_using_role-based_access_control)
+   for more information.
+
+## Pre-requisites
+
+In order to install the Tekton Dashboard, please make sure the following requirements are met:
+- A [kubernetes cluster](https://github.com/kubernetes/kubernetes) must be installed with version 1.15.0 or later.
+- [Tekton Pipelines](https://github.com/tektoncd/pipeline) must be installed in the cluster.
+  Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
+- Optionally, install [Tekton Triggers](https://github.com/tektoncd/triggers) if you want to manage triggers resources from the dashboard.
+Instructions to install Tekton Triggers can be found [here](https://github.com/tektoncd/triggers/blob/master/docs/install.md).
+
+## Which version should I use
+
+Every Tekton Dashboard version is meant to support specific Tekton Pipelines and Tekton Triggers versions.
+See the [docs on the main page](../README.md) to find the Tekton Dashboard version that suits your needs.
```

Comment:
This link will need to change when we start publishing this content on the website. Either we move the compatibility matrix here (which means we need to duplicate part of it to have easy links to different versions / docs on the README for folks coming to the repo) or we change this to an absolute link to the GitHub repo.

Reply:
I will take care of the website in another PR, i need to generate the website locally to experiment with it.
I would like to avoid duplication and i find it makes sense to show the compatibility matrix in the main README.

Reply:
let's ignore the website for now, potential changes coming (new design proposal opened last night) there that will impact how we structure / surface the docs. Let's get our content in first and deal with the website later

Reply:
makes sense